hpct-ldap-server/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+# This file populates the Overview on Charmhub.
+
+# The charm package name, no spaces (required)
+# See https://juju.is/docs/sdk/naming#heading--naming-charms for guidance.
+name: openldap-server
+display-name: openldap-server
+description: OpenLDAP server operator for HPC operations.
+summary: This operator is responsible for the OpenLDAP server and replicas.
+
+peers:
+  replicas:
+    interface: ldap-replicas
+provides:
+  ldap-auth:
+    interface: ldap-auth
```

Comment:
Add newline at end of file to avoid warning.

Reply:
Updated as suggested